packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/About.1.tool.mdx

```diff
@@ -445,3 +445,107 @@ it('should update an application's locale, () => {
   Terra.setApplicationLocale('en');
 });
 ```
+
+## Remote Screenshots Migration
+
+<Notice variant="important" ariaLevel="3">
+
+  The Remote Screenshots Mirgation feature is currently only supported by internal projects built using Jenkins at Cerner Corporation. Open source projects that utilize other CI such as Travis are not supported.
+
+</Notice>
+
+WebdriverIO reference screenshots used for screenshot comparisons during testing are stored in the `tests/wdio/__snapshots__/reference` folder. Large projects with many test steps require capturing a large set of screenshots. Storing a large set of screenshots locally in the project will greatly increase the size of the project and can degrade performance when cloning and building the project.
+Terra Functional Tessting offers this feature to migrate all reference screenshots stored in your project to a remote repository named Nexus. As a result, screenshots will no longer be stored in the project and committed to github. This will reduce the project size and improve performance.
+
+### Build Workflows
```

Comment:
It would be nice to add some flow diagrams to illustrate these different build workflows but I ran out of time. I'll add that as a todo for myself next time.